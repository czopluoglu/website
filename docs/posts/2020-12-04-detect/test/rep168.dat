10001001111011001110
00001011010111111110
10100001111001111110
10010001101000010010
11110001001000101010
10100111100110011100
00100000010010000111
11011100111001111011
01111111011011110010
11000101001111001111
00001101101010111111
11110111011111111111
10000000011111111010
10111010001010100111
01101000001000101011
00001111111110111110
11011011111000100010
10111010111110101011
11000001000000110110
10001001111001010100
10011111011111111111
01000000001110100111
01110111011100100110
11111111101111101111
01001101011111100100
01100000011001100110
11101011001001100110
10010101101100110011
11000011111010011111
01011111110111111111
00110111111001101010
01011001011101110110
11111111111111101111
10100011011111100110
00011111011100011111
00000111101111111111
10010111111010010110
10111100011000101100
00001010000110011110
11000111111110011001
01100101111101111110
11001111011101001001
01101111111111110001
10101001111111111001
11110000011111010101
11000101001110111111
11111000100110100111
11101101011001110011
00000001111011111110
11100111110000000011
01111000011010010111
11111000101111111010
11000001100111111000
11001101101100101110
11111101011010111100
10111101111011101111
10111111101111110101
10111001011011100010
01111101000010111011
01110101111110001111
11110101111110010011
00000001101110000101
10110101111011011111
10110100011111000100
11000001011000011000
10101100010011110011
10110101111011001100
11010101110000100000
00001101101111101111
00111001111111100100
10111110111110011111
00100000001010001111
00000001001000000101
01000111110000001001
11111111111111111110
00110000001111100101
11100111111111110100
10000101001010000000
01111011001100010111
11101111111010111111
11111101111110101100
11011001111001000010
01010100101010010110
10010011111001101011
10111101111110010110
11111101111111101110
01000001101101100011
01111000001001110001
10111011110111101110
11111101011110011111
10011011100000010111
01110001100101111111
11100111011000101010
00101011000011100111
00001001000110011011
11101101111111101111
01101111011100111111
10001001001010100100
01100000000011000000
01111000001010101111
00110101111110111011
00110101011101111011
11110001111000111111
11010100011010101111
10100000111111110111
00001101111011000111
11111111100111010100
00100000000100101010
00001101110001111100
01000001001000000010
11100001001010010000
01011111111010011100
10010010111110001111
10111100111100001101
10010001011111100111
00100001101011101000
00010001001000011110
01000100101010101110
01110100110111001111
01010000000010001011
00111110001110110110
01000100000100011111
10100001101111111111
10011101111101100111
11000001001110010001
11100110111011001111
11111111000010000110
11111011111110101111
11110001111110111111
11101111011111011111
01011100011111111111
11000111101101100010
00000001001110001111
10111000110000001000
01000000111101111101
00010000001100010010
11001000010011111111
11010011011110001011
00100100111011111111
10100000111010110011
10000001011000000100
01000000001111010111
01000001101101110111
00110111011000100011
11011101111100101110
10100101101101010011
01010110101000010111
10000000011100000010
00000001000010000110
11011111111110111101
11101100100111110011
00100001100110000100
11010100111010001110
00001010001011100111
10001001001100111000
11011001001111111011
11010001011101101111
10101001000110100101
11000110101000110110
01011111111110111111
00010101111001111010
11100111111111101110
11101000101101100110
11010000110100101010
11110111111000010011
11010000001000001110
10111100111111011110
01101100011010110011
10010110110111110011
01111000101010111011
10010100001101111011
11011100000100111111
01000101000000101011
01010101011010011100
10110101011010111111
10001001011110111111
11111111110110101011
01000010110100001110
00011100001111111111
11000101000000000011
01000000111011001110
00010111101100111101
01000100100001101010
11111011111010110010
10011111001010110111
11110000111111110011
10000100101100001000
10110100011110111010
11101101101010110110
00001011101110100110
11111000110001110101
10110111011001100010
11111011101001111110
10000000001111101111
00111101111111110011
10011101110110110111
11011001111111001010
10101101101111001111
00110001100111100010
11000001110011110111
01100100001001011101
11000100001110000010
11000111011011111011
11111111001101111111
10110100101110011111
10101001001111111010
10100111011100010011
11101111111111111111
10011100001100100110
00001101001011110111
01100111110110001110
01100111100110011111
01110001000111101100
10001101110010111011
00101101000000100100
11000011101111101110
10100111001110010110
10101010111111111111
10110101001110100010
01001100111101100001
01000101111100000110
00100111011010000001
01101111111101100010
01000000001100010011
00000011111111000001
10011010111010001101
11000101000111101111
11110111111010000110
01011001110000000111
11000001110010110000
01010011001100101110
00010101111000000110
01000101101011101011
11010001111010110110
00110011011110110110
11111110101011101111
11110001011100101101
10000101111001110110
11001100110011110111
00001001100100100111
01111000001100101101
10100001110011111011
11110011111111110111
11010011111100110011
10110100001111011111
01010100001101101110
01100011001100111110
00000001100000001011
10010001111000000010
11100001001110110101
