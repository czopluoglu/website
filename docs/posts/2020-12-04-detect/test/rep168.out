-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep168.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.80

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.88
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  6
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7070
                   IDN index value:   0.7526
                           Ratio r:   0.6338

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    3    4    3    1    1    1    1    1
    4    5    5    3    3    6    5    5    6    6

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    6    7    8    9   10

   -----------CLUSTER  2-------------

    2

   -----------CLUSTER  3-------------

    3    5   14   15

   -----------CLUSTER  4-------------

    4   11

   -----------CLUSTER  5-------------

   12   13   17   18

   -----------CLUSTER  6-------------

   16   19   20
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d--+++-+---++-+-----
   -d+++++----++--+-+--
   -+d+++++----------+-
   +++d++----+---------
   ++++d+---+----------
   +++++d-+-+-++-------
   -++---d---++---+--+-
   +-+--+-d++++--+-----
   -------+d+++--+-+--+
   ----++-++d+--+++---+
   ---+--++++d--+---++-
   ++---++++--d+-----+-
   ++---+-----+d+--+---
   ---------++-+d+++--+
   +------+++---+d-++-+
   -+----+--+---+-d+-++
   --------+---++++d---
   -+--------+---+--d-+
   --+---+---++---+--d+
   --------++---+++-++d
 
 No cross validation for this DETECT run
